.env.example

```diff
@@ -0,0 +1,18 @@
+# This application uses a single environment file, as several fields are used by both the API and UI
+
+# Message one of the Team Admins to get access to this - make sure it's never committed!
+DISCORD_CLIENT_ID = CLIENT_ID
+DISCORD_CLIENT_SECRET = CLIENT_SECRET
+
+# Basic defaults for a MongoDB image
+DATABASE_USER = root
+DATABASE_PASSWORD = example
+DATABASE_HOST = db
+DATABASE_PORT = 27017
+
+# For security reasons, Vite won't include environment variables in the front-end build unless it's prefixed with VITE_
+VITE_API_URL = http://localhost:8080
+VITE_SENTRY_DSN = ...
```

Comment:
```suggestion
# VITE_SENTRY_DSN = ...
```

I think it'd be good to default to Sentry being disabled by default in development environments